go/exec/exec.go

```diff
@@ -156,9 +167,17 @@ const (
 
 // NewExec creates a new *Exec with an autogenerated uuid.UUID as well
 // as a constructed infra.Infra.
```

Comment:
I guess that means this comment can be updated with "autogenerated uuid.UUID OR provided UUID"? (If I understand correctly)